plugins/rapid7_insightvm/komand_rapid7_insightvm/actions/update_shared_credential/action.py

```diff
@@ -0,0 +1,171 @@
+import insightconnect_plugin_runtime
+from insightconnect_plugin_runtime.exceptions import PluginException
+from .schema import UpdateSharedCredentialInput, UpdateSharedCredentialOutput, Input, Output, Component
+from ...util import endpoints
+from ...util.resource_requests import ResourceRequests
+from ...util.util import check_not_null, check_in_enum, make_payload
+
+
+def as400_cifs_cvs(account: dict):
+    service = account.get("service")
+    domain = account.get("domain", "")
+    username = check_not_null(account.get("username"))
+    password = check_not_null(account.get("password"))
```

Comment:
Username & Password seem to be very commonly occurring fields between functions.
Maybe we could do something like
```
def user_and_pw(username, pw):
    if username and pw:
        return check_not_null(account.get("username")), check_not_null(account.get("password"))
    else:
        return check_not_null(account.get("username"))

Reply:
And then retrieve the variables like this
```
(username, password) = user_and_pw(username, pw)
```

Reply:
smort, i will do exactly that